package.json

```diff
@@ -50,14 +50,13 @@
   "dependencies": {
     "azure-pipelines-task-lib": "2.9.3",
     "jquery": "^3.4.1",
-    "request": "^2.88.0",
```

Comment:
I removed this because it has a vulnerability.  I couldn't find anywhere that it was being used.